Rocket.Chat/Controllers/Chat/ChannelInfoViewController.swift

```diff
@@ -22,6 +23,14 @@ class ChannelInfoViewController: BaseViewController {
         didSet {
             guard let subscription = self.subscription else { return }
 
+            #if DEBUG
+                let flex = [ChannelInfoDetailCellData(title: localized("Open FLEX"), detail: "", action: {
```

Comment:
1. String is not localized (and doesn't need to be), just use a String and add **DEBUG: Open FLEX**;
2. Let's also add this on **BETA**;
3. Let's move it to Settings;